examples/ptr_inversion/material_obj/forward_and_adjoint.i

```diff
@@ -0,0 +1,243 @@
+[Mesh]
+  [generated]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 25
+    ny = 25
+    ymin = 0
+    xmin = 0
+    xmax = 10
+    ymax = 10
+  []
+  # coord_type = RZ
+[]
+
+[Problem]
+  nl_sys_names = 'nl0 adjoint'
+  kernel_coverage_check = FALSE
+[]
+
+[Variables]
+  [T_real]
+    initial_condition = 1e-8
+  []
+  [T_imag]
+    initial_condition = 1e-8
+  []
+  [T_real_adj]
+    initial_condition = 1e-8
+    solver_sys = adjoint
+  []
+  [T_imag_adj]
+    initial_condition = 1e-8
+    solver_sys = adjoint
+  []
+[]
+
+[Kernels]
+  [heat_conduction_real]
+    type = ADMatDiffusion
+    variable = T_real
+    diffusivity = k
+  []
+  [02596*]
```

Comment:
why this weird block name

Reply:
Most likely I overwritten the actual block name at some point and didn't catch the error.

Reply:
Fix this since you have to add the gold file anyway